controller/controlplane/controller.go

```diff
@@ -373,7 +373,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 			k8sutils.NewConditionWithGeneration(
 				ConditionTypeWatchNamespaceGrantsValid,
 				metav1.ConditionFalse,
-				kcfgcontrolplane.ConditionReasonMissingReferenceGrant,
+				kcfgcontrolplane.ConditionReasonWatchNamespaceGrantInvalid,
```

Comment:
This (as well as condition definition above and used conditions below) is handled in https://github.com/Kong/gateway-operator/pull/1494/files#diff-64ee605326f88832bc39c8e4902efd8f4fe1c5072d0d6fa5476cf4d1f113f102.

I'm fine leaving this as is but we can also leave #1494 to handle it.

Reply:
Good to know, I needed to change it as I had to bump kconf. Let's wait for #1494 to be merged first 👍🏻